ruff.toml

```diff
@@ -0,0 +1,9 @@
+line-length = 120
+target-version = "py310"
+
```

Comment:
Do we want to exclude all worlds? We can add `extend-exclude = ["worlds"]` to add it. If we want to include some limited set of rules for worlds we should be able to stick another ruff.toml in that dir and it'll override the settings.